lib/hash.cc

```diff
@@ -293,15 +267,14 @@ static int hash_primes[] = {
     65357
 };
 
-int
-hashPrime(int n)
-{
+uint32_t
+hash_table::hashPrime(uint32_t n) {
     int I = sizeof(hash_primes) / sizeof(int);
     int best_prime = hash_primes[0];
-    double min = fabs(log((double) n) - log((double) hash_primes[0]));
+    double min = fabs(log((double)n) - log((double)hash_primes[0]));
     double d;
     for (int i = 0; i < I; ++i) {
-        d = fabs(log((double) n) - log((double) hash_primes[i]));
+        d = fabs(log((double)n) - log((double)hash_primes[i]));
```

Comment:
Same unnecessary whitespace change here.
```suggestion
        d = fabs(log((double) n) - log((double) hash_primes[i]));
```